bin/extract-strings.mjs

```diff
@@ -50,10 +83,10 @@ const OUTPUT_BANNER = `#
 
 /** @type {import("i18next-parser").UserConfig} */
 const config = {
-  // Our translation system requires that we add ".zero", ".one", ".other" keys for plurals
+  // Our translation system requires that we add all 6 forms (zero, one, two, few, many, other) keys for plurals
   // i18next-parser extracts plural keys based on the target locale, so we are passing a
-  // locale that need exactly the ".zero", ".one", ".other" keys, even if we are extracting English strings
-  locales: ["lv"],
+  // locale that need exactly the 6 forms, even if we are extracting English strings
+  locales: ["ar"],
```

Comment:
I wonder why make ar the default instead of en

Reply:
It is explained in the comment above the line:
- our translation system requires us to define 6 keys for plurals (`[key].zero`,`[key].one`, `[key].two`, `[key].few`, `[key].many`, `[key].other`)
- the tool we use (`i18next-parsers`) extracts these plural keys based on the source locale. Since English has only `one` and `other` forms, if we set `en` as the source locale it would extract only `[key].one` and `[key].other`
- `ar` has all six forms, so setting it as the source locale has the effect of letting the tool extract all the needed strings